modules/ROOT/edits/uninstall-service-mesh.adoc

```diff
@@ -0,0 +1,20 @@
+= Uninstall Service Mesh
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, uninstalling
+
+You might need to uninstall or delete Anypoint Service Mesh: for example, to troubleshoot issues. If you uninstall Anypoint Service Mesh and it includes created bindings or adapters, or if you created an application with an injected Envoy, the application stops working. The command removes all the Kubernetes components associated with the Anypoint Service Mesh project.
```

Comment:
Also, which command? There is a lot of text between here and the end.
I think we might be saying that before you uninstall, you must meet the following prereqs.
So, consider this:
```suggestion
You might need to uninstall or delete Anypoint Service Mesh: for example, to troubleshoot issues. Before you uninstall or delete, you must take the following precautions to avoid disabling your application: 
```

Reply:
Done! Thanks!